core/cmd/remote_client_test.go

```diff
@@ -274,6 +278,7 @@ func TestClient_RemoteLogin(t *testing.T) {
 
 			set := flag.NewFlagSet("test", 0)
 			set.String("file", test.file, "")
+			set.Bool("bypass-version-check", true, "")
```

Comment:
Have to set this flag on all tests that do login since version is unset in tests